README.md

```diff
@@ -7,31 +7,33 @@ The GSLab Template is a minimal working demonstration of the tools and organizat
 You'll need the following to run the template. [Homebrew](https://brew.sh/) for Mac and [Linuxbrew](http://linuxbrew.sh/) for Linux make this easier.
 * Windows `cmd.exe`, Mac OS X `bash`, or Linux `bash`. 
 * [Python 2.7.X](https://wiki.python.org/moin/BeginnersGuide/Download) for [Windows](https://docs.python.org/2/using/windows.html), [Mac](https://docs.python.org/2/using/mac.html) or [Linux](https://docs.python.org/2/using/unix.html).
-    * [gslab_python](https://github.com/gslab-econ/gslab_python) version 4.1.0 or later.
-    * [PyYAML](http://pyyaml.org/wiki/PyYAML) a Python module for parsing YAML files. 
 * [git](https://git-scm.com/downloads) for version control.
     * [git-lfs](https://git-lfs.github.com/) for versioning large files. 
     * You'll need both git and git-lfs to clone the repository. 
 * [LyX](https://www.lyx.org/Download) (with instructions for LaTeX) 
     * Add LyX to your PATH for [Windows](http://www.computerhope.com/issues/ch000549.htm), [Mac](http://hathaway.cc/post/69201163472/how-to-edit-your-path-environment-variables-on-mac), and [Linux](http://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux).
     * The beamer theme [`metropolis`](https://github.com/matze/mtheme). This is part of MikTeX since Dec 2014. 
 
-## Getting started
+## Quick start
 
 1. Open a shell, clone the repository, and navigate to its root.
     ```bash
     git clone https://github.com/gslab-econ/template.git
```

Comment:
I don't know where we changed cloning protocol from SSH to HTTPS, but we should be consistent (with the `Copying the template` bit.

Reply:
I've also had issue cloning the repo. Let's have a bit in FAQ saying "If you have this error then try `git lfs clone`."

```bash
Errors logged to /Users/lequan/ssh_template/.git/lfs/objects/logs/20170904T151244.091215032.log
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs smudge -- %f' failed 2
error: external filter 'git-lfs smudge -- %f' failed
fatal: release/paper/text.pdf: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'

```

Reply:
Updated default to  
```
git lfs clone
```